src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,149 @@
+package wolox.training.models;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
```

Comment:
Generate a field called id of type long that is autoincrementing. Don't forget to add the @Id annotation for this field.